vagrant/README.md

```diff
@@ -1,3 +1,36 @@
-Vagrant VM to build and run Heron
-=================================
-vagrant up
\ No newline at end of file
+# Vagrant image for running Heron examples
```

Comment:
Should be a bit more specific to emphasize that it focuses on Kafka-based topologies. It seems that overview also needs to be changed in this regard